src/uistate.h

```diff
@@ -13,6 +13,32 @@
 
 class item;
 
+struct comestible_inv_save_state {
+    public:
+        int sort_idx = -1;
+        std::string filter = "";
```

Comment:
```suggestion
        std::string filter;
```
Redundant initialization.